extensions/Worker.Extensions.SignalRService/src/SignalRServiceDependencyInjectionExtensions.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Azure.Functions.Worker.SignalRService;
+using Microsoft.Extensions.Azure;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+using Microsoft.Extensions.Hosting;
+
+namespace Microsoft.Extensions.DependencyInjection
+{
+    public static class SignalRServiceDependencyInjectionExtensions
+    {
+        public static IServiceCollection AddServerlessHub<THub>(this IServiceCollection services) where THub : ServerlessHub
```

Comment:
Something to consider is also adding a `AddServerlessHub(this IServiceCollection services, Type hubType)` - and verifying that `hubType` is an expected `ServerlessHub` implementation. This is a small convenience API for when customers are doing some more advanced scenario where they are resolving their serverless hub types at app startup.